RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -1106,6 +1106,10 @@ boolean L12_sonofaBeach()
 	{
 		return false;
 	}
+	if(ed_DelayNC(100.0))	//zerg rush can deal 100% of maxHP in damage
+	{
+		return false;	//ed is not prepared. delay
+	}
```

Comment:
This could infinitely prolong clearing the beach until there's nothing else left to do.
I don't think this is a good solution to this. At the point we're doing the level 12 quest Ka and thus linen bandages should be abundant and if not the cost of a single adventure to visit the underworld is better than stalling out the run indefinitely.

Reply:
1. It specifically checks for linen bandages and does not delay if we have linen bandages.
2. It specifically checks if we are about to powerlevel so it does not stall indefinitely.